src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace GitHook\Command\FileCommand\PreCommit\CodeStyleSniff;
+
+use GitHook\Command\Context\CommandContextInterface;
+use Symfony\Component\Yaml\Parser;
+
+class CodeStyleSniffConfiguration
+{
+    /**
+     * @var string
+     */
+    protected const MODULE_CONFIG_TOOL_KEY = 'code-sniffer';
+
+    /**
+     * @var string
+     */
+    protected const CORE_MODULE_PATH_REGEX = '/.*\/vendor\/spryker\/.*\/Bundles\/.*\/src\//';
```

Comment:
Do we need this one? It also doesnt seem to work for spryker-shop or other Namespaces

Reply:
It matches changed module to get its `tooling.yml`.
This regex works with any `spryker/*` package

Reply:
Well, I guess any nonsplit
my main concern is the hardcoding of nonsplit Bundles/ stuff that is only visible and useful Spryker internally (never externally on split).

But fair enough for now then.

Reply:
```suggestion
    protected const CORE_MODULE_PATH_REGEX = '/.*\/vendor\/spryker\/.+\/Bundles\/.+\/src\//';
```
I dont see how it is useful to use any when it must be at least a non empty string.

Reply:
Also, use # as regexp delimiter, then you dont need to escape normal forward slash.